transport/transporters/rabbitmq/config.go

```diff
@@ -0,0 +1,74 @@
+/*
+  Copyright 2019 Nextdoor.com, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package rabbitmq
+
+import (
+	cli "gopkg.in/Nextdoor/cli.v1"
+	"gopkg.in/Nextdoor/cli.v1/altsrc"
+)
+
+const (
+	ConfVarExchangeName   = "exchange-name"
+	ConfVarUsername       = "rabbitmq-username"
+	ConfVarPassword       = "rabbitmq-password"
+	ConfVarHost           = "rabbitmq-host"
+	ConfVarVirtualHost    = "rabbitmq-vhost"
+	ConfVarWriteBatchSize = "rabbitmq-batch-size"
+)
+
+var Flags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "config",
+		Value: "config.yaml",
+		Usage: "bifrost YAML config file",
+	},
+	altsrc.NewStringFlag(cli.StringFlag{
+		Name:   ConfVarExchangeName,
+		Usage:  "RabbitMQ Exchange Name",
+		EnvVar: "BIFROST_RABBITMQ_EXCHANGE_NAME",
```

Comment:
Are there more standard RabbitMQ variables we can use here - rather than Bifrost-specific ones? As an example, for Kinesis we use the standard AWS variables (https://github.com/Nextdoor/pg-bifrost/blob/cabf3e6df5d33a7a7a719adcf6b12684311b869f/transport/transporters/kinesis/config.go#L46).

Reply:
It looks like the AMQP package supports `AMQP_URL`. Perhaps it would be better to simply do that, than have all these different variables?

Reply:
https://github.com/streadway/amqp/blob/e5adc2ada8b8efff032bf61173a233d143e9318e/doc.go#L73

Reply:
We could do that.